pkgs/development/python-modules/flask-principal/default.nix

```diff
@@ -1,7 +1,7 @@
 { lib, buildPythonPackage, fetchPypi, flask, blinker, nose }:
 
 buildPythonPackage rec {
-  pname = "Flask-Principal";
+  pname = "flask-principal";
```

Comment:
Can you double check if keeping the `pname` lowercase still lets us download from PyPI (changing the hash to `lib.fakeHash` to force a download)? It fails for me here, even though the main page of this package on PyPI works regardless (flask-principal 301 redirects to Flask-Principal).

Reply:
Thanks!
You are right, it seems that the cache has remained.
I also updated the hash to the SRI hash.